pkg/admission/customresourcevalidation/rolebindingrestriction/validation/validation.go

```diff
@@ -0,0 +1,113 @@
+package validation
```

Comment:
I would expect this to be a move with minor type changes.

Reply:
Also did we not have unit tests?

Reply:
Yes, it is, and with only RBR types, not the other auth types.

Reply:
unit tests here: pkg/authorization/apiserver/admission/restrictusers/restrictusers_test.go

Reply:
(you should pull them to this cr validation, then)